msteams-platform/task-modules-and-cards/task-modules/task-modules-bots.md

```diff
@@ -1,33 +1,36 @@
+# Use task modules from bots
+
+Task modules can be invoked from Microsoft Teams bots using buttons on Adaptive Cards and Bot Framework cards that is hero, thumbnail, and Office 365 connector. Task modules are often a better user experience than multiple conversation steps where as a developer you have to keep track of bot state and allow the user to interrupt or cancel the sequence.
 
 There are two ways of invoking task modules:
 
-* **A new kind of invoke message `task/fetch`.** Using the `invoke` [card action](~/task-modules-and-cards/cards/cards-actions.md#invoke) for Bot Framework cards, or the `Action.Submit` [card action](~/task-modules-and-cards/cards/cards-actions.md#adaptive-cards-actions) for Adaptive cards, with `task/fetch`, the task module (either a URL or an Adaptive card) is fetched dynamically from your bot.
-* **Deep link URLs.** Using the [deep link syntax for task modules](~/task-modules-and-cards/what-are-task-modules.md#task-module-deep-link-syntax), you can use the `openUrl` [card action](~/task-modules-and-cards/cards/cards-actions.md#openurl) for Bot Framework cards or the `Action.OpenUrl` [card action](~/task-modules-and-cards/cards/cards-actions.md#adaptive-cards-actions) for Adaptive cards, respectively. With deep link URLs, the task module URL or Adaptive card body is obviously known in advance, avoiding a server round-trip relative to `task/fetch`.
+* A new kind of invoke message `task/fetch`: Using the `invoke` [card action](~/task-modules-and-cards/cards/cards-actions.md#invoke-action-type) for Bot Framework cards, or the `Action.Submit` [card action](~/task-modules-and-cards/cards/cards-actions.md#adaptive-cards-actions) for Adaptive cards, with `task/fetch`, the task module either a URL or an Adaptive Card is fetched dynamically from your bot.
+* Deep link URLs: Using the [deep link syntax for task modules](~/task-modules-and-cards/task-modules/invoking-task-modules.md#task-module-deep-link-syntax), you can use the `openUrl` [card action](~/task-modules-and-cards/cards/cards-actions.md#openurl-action-type) for Bot Framework cards or the `Action.OpenUrl` [card action](~/task-modules-and-cards/cards/cards-actions.md#adaptive-cards-actions) for Adaptive cards, respectively. With deep link URLs, the task module URL or Adaptive Card body is obviously known in advance, avoiding a server round-trip relative to `task/fetch`.
+
+> [!IMPORTANT]
+> To ensure secure communications, each `url` and `fallbackUrl` must implement the HTTPS encryption protocol.
 
->[!IMPORTANT]
->To ensure secure communications, each `url` and `fallbackUrl` must implement the HTTPS encryption protocol.
+The next section provides details on invoking a task module using `task/fetch`.
 
-## Invoking a task module via task/fetch
+## Invoke a task module using task/fetch
```

Comment:
Rewrite all headings without /.
This can be:
```suggestion
## Invoke a task module using task or fetch
```

Reply:
Its a name in code cannot be changed